docs/sop/RESTORE_DATABASE_AWS_BACKUP.md

```diff
@@ -54,7 +56,7 @@ the cheapest instance type `db.t3.micro`
 9. In the *Connectivity* box, select the required VPC based on `live` 
 environment, i.e. `vpc-ap-east-1-live-gigadb-gigadb`
 10. In the *Connectivity* box, `Public access` should be *No*
-11. In the *Connectivity* box, select VPC security group, e.g. `rds_sg_live_gigadb-20220426875429729`
+11. In the *Connectivity* box, select VPC security group, e.g. `rds_sg_live_gigadb-*`
 12. In the *Connectivity* box, *Password authentication* should be deleted
```

Comment:
Hi @pli888,

*Password authentication* cannot be found in the *Connectivity* box, as below:
![image](https://github.com/gigascience/gigadb-website/assets/64770635/f0a59ec0-3554-4363-ac9a-dc5dd1bd7853)